hydrogym/core.py

```diff
@@ -292,6 +298,43 @@ def solve(
             PDEBase: The state of the PDE at the end of the solve
         """
     for iter, t in enumerate(np.arange(*t_span, self.dt)):
+      iter = iter + start_iteration_value
```

Comment:
It looks like `solve` and `solve_multistep` do almost the same thing. For a "public" release It may be better to merge them.:)

Reply:
I agree - what's the distinction supposed to be here?